CHANGELOG.md

```diff
@@ -11,15 +11,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Fix incorrect type definitions in date formatting.
+- Fix the Jest configuration to not use the deprecated 'tsConfig' spelling of 'tsconfig'.
 - Fix removeEventListener bug for WithTooltip.
 - Reset `isAudioOn` state to `true` when `LocalAudioOutputProvider` unmounts.
 
 ### Added
 
+- - Added some key properties to 'ModalButtonGroup', which quiets some React warnings.
 - Added an optional prop `onPopOverClick` to pass a callback function to the `PopOver` UI component. This callback will be called when the `PopOver` UI component is clicked.
 - Add `--no-fail-on-empty-changeset` flag in deploy script to not fail for empty changeset
 
 ### Changed
+- Bumped react and react-dom version to 17.
+- Updated versions of testing-library family of packages
```

Comment:
Nit: Could you please add a "." at the end of sentence here?

Reply:
Yes, will do.